RecoCTPPS/PixelLocal/src/RPixDetClusterizer.cc

```diff
@@ -64,11 +64,11 @@ void RPixDetClusterizer::buildClusters(unsigned int detId, const std::vector<CTP
     if(!is_in){
 //calibrate digi and store the new ones
       electrons = calibrate(detId,adc,row,column,pcalibrations);
+      if(electrons < SeedADCThreshold_*ElectronADCGain_) electrons = SeedADCThreshold_*ElectronADCGain_;
```

Comment:
> The second problem is something similar to what already discussed for the central pixel code last week between Danek and Andrea.

this is not exactly the same.
There we have an override to calibrated digis loaded per pixel, but that override is still below the seed threshold.
Here, apparently there is no distinction between a seed and a regular pixel thresholds.

Reply:
It may also make sense to rename the threshold to ```seedADCLowerBound_``` because it's not serving as a threshold anymore.

BTW, I noticed that in the original review we missed the CMS style violations for the variable names, which should be starting with lower case.
This cleanup could be made in a follow up PR.